tests.rkt

```diff
@@ -74,3 +75,25 @@ for (int i = 0; i < 10; i++) {
 END
                 ))
 
+(test-case "overflow on subsequent line does not backtrack"
```

Comment:
Note that this test would pass the initial version of `pprint` at http://planet.racket-lang.org/package-source/dherman/pprint.plt/1/0/pprint.ss

It's a regression in the second version.